tests/test_users.py

```diff
@@ -80,31 +82,30 @@ def test_get_user_success(mocker):
     artifactory_user.get.assert_called_once()
 
 
-@responses.activate
-def test_list_user_success(mocker):
-    responses.add(
-        responses.GET,
-        f"{URL}/api/security/users",
-        json=[SIMPLE_USER.dict()],
-        status=200,
-    )
+# Disable because mock can't serialize pydantic2 HttpUrl

```

Comment:
No need to disable this just replace the mock by:

```
responses.add(
        responses.GET,
        f"{URL}/api/security/users",
        json={"name":"test_user", "uri":"https://some.uri"},
        status=200,
    )
```

Reply:
Or even better:

```
responses.add(
        responses.GET,
        f"{URL}/api/security/users",
        json=[json.loads(SIMPLE_USER.model_dump_json())],
        status=200,
    )
```

Reply:
done